contracts/Ante_DaiPegTest.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+import "./interfaces/AnteTest.sol";
+
+// Ante Test to check DAI remains +- 5% of USD
+contract Ante_DaiPegTest is AnteTest("DAI is pegged to USD") {
+    // https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f
+    address public constant DaiAddr = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+
+    AggregatorV3Interface internal priceFeed;
+    /**
+     * Network: Mainnet
+     * Aggregator: DAI/USD
+     * Address: 0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9
+     */
+    constructor() {
+        protocolName = "DAI";
+        testedContracts = [DaiAddr];
+        priceFeed = AggregatorV3Interface(0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9);
```

Comment:
A point of consideration from our lead dev:
Not sure how robust the chainlink oracle is to flash loans, probably want a TWAP vs just a single price check? What do you know about oracles and are there other places to reference the pricefeed from?

Reply:
Chainlink use a (mostly) VWAP, so it responds faster than TWAPs. It also fetches data from multiple off-chain sources so it isn't succeptible to flash loan attacks. Each Chainlink node can decide how they calculate price (TWAP, VWAP, etc) so the price feed is resilient unless the nodes collude. If you'd rather use a TWAP, Andre Cronje developed this: https://feeds.sushiquote.finance/ which I believe uses DEX TWAPs. Let me know what you think!

Reply:
I think that works out well! I'll resolve this now!